eclipse-ee4j/jaxrs-api
# TCK Migration: Move jaxrs/spec tests from jakartaee-tck

This change is in continuation of TCK migration from https://github.com/eclipse-ee4j/jaxrs-api/pull/1002 .

 To test the changes :

export JAVA_HOME=<JDK11_HOME>
export M2_HOME=<MAVEN_HOME>
export PATH=$M2_HOME/bin:$JAVA_HOME/bin:$PATH

jaxrs-tck/
mvn clean install

jersey-tck/
run all tests : `mvn clean verify -Parq-glassfish-managed`
skip optional tests : `mvn clean verify -Parq-glassfish-managed -DexcludedGroups="xml_binding"`  

The tests migrated :
jakartaee-tck/src/com/sun/ts/tests/jaxrs/spec(317)

Current status : 
1 test failure in jakarta.ws.rs.tck.spec.provider.overridestandard.JAXRSClientIT.readWriteStreamingOutputProviderTest is commented for now. This will be tracked at https://github.com/eclipse-ee4j/jaxrs-api/issues/1020

2 tests are disabled as it was in jakartaee-tck:
jakarta.ws.rs.tck.spec.resource.responsemediatype.JAXRSClientIT.noProducesTest
jakarta.ws.rs.tck.spec.resource.responsemediatype.JAXRSClientIT.noProducesResponseReturnTest
